src/app.js

```diff
@@ -122,7 +123,19 @@ class App {
                 await this.db.sync();
             }
             //log configuration data
-            this.logger.debug(JSON.stringify(this.config.getConfigurationInfo()));
+            const userConfig = this.config.getConfigurationInfo();
+            this.logger.debug(JSON.stringify(userConfig));
+            //check important change of configurations
+            const res = await this.db.queries.getConfiguration();
+            if(res !== null) {
+                const dbuserConfig = JSON.parse(res.config)
+                if(dbuserConfig.LISTEN_MODE !== userConfig.LISTEN_MODE && userConfig.LISTEN_MODE == 1) {
+                    this._needResync = true;
+                    this.logger.error(`ATTENTION: LISTEN_MODE changed from the last boot, please resync the database`);
```

Comment:
This is imo the wrong check.  
What we actually want to check is if the set of tokens to monitor has changed. This may or may not be caused by LISTEN_MODE having changed.   
I also suggest to rename LISTEN_MODE to something like ONLY_LISTED_TOKENS (inverted logic - so it would become false by default).